### PR TITLE
Fix the search functionality of the website & add a search icon to th…

### PR DESCRIPTION
**Pull request by nobledeveloper01**
…e search input field #179

### CODE REVIEW

File: src/index.html
Comment (OSCA-Ado-Ekiti) [inline src/index.html:1]: @nobledeveloper01 The meta tags in this file are for SEO purpose to help the site to be visible when people search for companies in Africa. I'm sorry to stress you, kindly help with adding them back

**Comment by OSCA-Ado-Ekiti** [comment]
@nobledeveloper01 Thank you for contributing, your PR is good to go. Just kindly help with adding those meta tags back so i can merge at once

**Comment by nobledeveloper01** [comment]
ok i will the meta tags gave some problems
i will uncomment them now

**Comment by Elijah699** [comment]
@nobledeveloper01 oh sorry to hear that, what problems please?

**Pull request merged**